Sources/TensorFlow/Loss.swift

```diff
@@ -13,6 +13,9 @@
 // limitations under the License.
 
 /// Returns the L1 loss between predictions and expectations.
+/// Given the x and y vectors L1 loss computes:
+///  `reduction([l_1, l_2, ..., l_N]^T)`, where `l_i = |x_i - y_i|`
```

Comment:
What does the `^T` represent?

Reply:
It's transpose, just to denote the shape of the vector